### ISSUE TITLE
[New Rule] Use comparision directly instead of bool(comparison)

### ISSUE THREAD

**Issue by MT-Cash** [MartinThoma/flake8-simplify#88]
## Explanation

Comparisons return boolean values. No need to wrap it in a bool-call

## Example

```python
# Bad
bool(a == b)

# Good
a == b
```

**Comment by MartinThoma** [comment]
* Experimental number: SIM901
* Permanent number: SIM224
* Release: not released so far
* End of trial period: 6 months after the release

**Comment by MartinThoma** [comment]
```
$ astpretty --no-show-offsets /dev/stdin <<< `cat /home/moose/example.py`

astpretty --no-show-offsets /dev/stdin <<< `cat /home/moose/example.py`
Module(
    body=[
        Expr(
            value=Call(
                func=Name(id='bool', ctx=Load()),
                args=[
                    Compare(
                        left=Name(id='a', ctx=Load()),
                        ops=[Eq()],
                        comparators=[Name(id='b', ctx=Load())],
                    ),
                ],
                keywords=[],
            ),
        ),
    ],
    type_ignores=[],
)
```
